jflex/src/main/java/jflex/CharClasses.java

```diff
@@ -120,11 +120,6 @@ public int getNumClasses() {
   public void makeClass(IntCharSet set, boolean caseless) {
     if (caseless) set = set.getCaseless(scanner.getUnicodeProperties());
 
-    if (DEBUG) {
```

Comment:
Right. Could the debugging be done on `System.out` rather than `Out`?

I was a bit unsure of the responsibiilty of Out. But because it's seems to have a state of the output (a count of warnings), it's not thread safe.

Reply:
Yes, that's a good point. I don't think it would be a problem to do the debug output on `System.out` instead.

Reply:
Done.